packages/babel-plugin-transform-regenerator/test/fixtures/arrow-function-inside-body/options.json

```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    "transform-regenerator"
```

Comment:
es2015 will transform arrow function, so there is no sense in this test. The goal is test for generator with arrow function inside transformation 🙂